strings.js

```diff
@@ -0,0 +1,32 @@
+const sayHello = (word) => {
+  return ('Hello, ' + word + '!');
```

Comment:
You could also use `template literals` to solve this. They are worth looking up as they are a lot more intuitive to work with.